frontend/src/pages/settings/account/Account.tsx

```diff
@@ -36,20 +36,39 @@ export const AccountPage = () => {
             xs={12}
             sm={12}
             md={9}
+            lg={9}
+            xl={10}
             spacing={3}
           >
-            <SettingsSection title={t('settings.changeEmailAddress')} xs={12}>
+            <SettingsSection
+              title={t('settings.changeEmailAddress')}
+              xs={12}
```

Comment:
Since we re-use `xs={12}`, `sm={12}`, etc. several times in this file, how about using a variable to store these values?

In the `AcountPage` component body we could have:
```javascript
const breakP = { xs: 12, sm: 12, md: 9, lg: 9, xl: 10 };
```
and in the returned JSX:
```jsx
xs={breakP.xs}
sm={breakP.sm}
md={breakP.md}
lg={breakP.lg}
xl={breakP.xl}
```

Reply:
> and in the returned JSX:
> 
> ```js
> xs={breakP.xs}
> sm={breakP.sm}
> md={breakP.md}
> lg={breakP.lg}
> xl={breakP.xl}
> ```

can also be written using the [spread syntax](https://reactjs.org/docs/jsx-in-depth.html#spread-attributes):
```jsx
<SettingsSection title="Title" {...breakP}>
```